setup.py

```diff
@@ -177,19 +178,22 @@ def get_build_options():
     # FIXME it is a wrong place for this dependency
     if not no_dist:
         include_dir_plat.append(mpi_root + '/include')
-    using_intel = os.environ.get('cc', '') in ['icc', 'icpc', 'icl', 'dpcpp']
+    using_intel = os.environ.get('CC', '') in [
```

Comment:
Are you confirmed change from lower to upper case didn't break anything?

Reply:
It's fine on windows side since I believe windows is case insensitive, can you confirm for linux @samir-nasibli?

Reply:
> Did you confirm change from lower to upper case didn't break anything?

Actually it fix building on both Win and linux, while we using intel compiler, since linux case sensitive. We just setting CC and CXX env vars before running setup.py